### DISCUSSION TITLE
chrome硬解视频

### DISCUSSION THREAD

**Discussion by WindDaoist** [linuxdeepin/developer-center#3315] [Q&A | 问答板块] [answered]
uos原生浏览器 是否有办法调用硬解播放视频

**Comment by luodeepin** [comment, marked as answer]
您好！
如果GPU驱动支持，并且主动按照chromium的硬解方式做了适配，在浏览器-设置-高级中开启 视频解码加速就可以了。
可以在 chrome://gpu 里面查看当期GPU是否适配！



If your computer's GPU driver has a compatibility processing process, and it can actively adapt to the browser through the hard solution of chromium, then you can follow the steps below:
Browser->Settings->Advanced->Enable video decoding acceleration

How to determine if a GPU is compatible?
Check whether the current browser version supports it directly through： chrome://gpu
